internal/airbyte/repository.go

```diff
@@ -104,6 +127,32 @@ func (r *Repository) ConnectionsCount() ([]ConnectionCount, error) {
 	return r.connectionCountQuery(query)
 }
 
+// ConnectionsLastSuccessfulSyncAge returns the age of the last successful sync job attempt
+// for active connections.
+func (r *Repository) ConnectionsLastSuccessfulSyncAge() ([]ConnectionSyncAge, error) {
+	// FIXME this query has horrendous performance!
```

Comment:
Tables and current attempts at optimizing this query:

- [last_successful_sync.sql.txt](https://github.com/botify-labs/airbyte_exporter/files/11760628/last_successful_sync.sql.txt)
- [tables.md](https://github.com/botify-labs/airbyte_exporter/files/11760629/tables.md)